docs/draft/howto/single-ownnamespace-install.md

```diff
@@ -0,0 +1,105 @@
+ - *MultiNamespace*: If supported, the operator can be configured to watch for events in more than one namespace.
+ - *AllNamespaces*: If supported, the operator can be configured to watch for events in all namespaces.
+
+OLMv1 will not attempt multi-tenancy (see [design decisions document](../../project/olmv1_design_decisions.md)) and will think of operators
+as globally installed, i.e. in OLMv0 parlance, as installed in *AllNamespaces* mode. However, there are operators that
+were intended only for the *SingleNamespace* and *OwnNamespace* install modes. In order to make these operators installable in v1 while they
+transition to the new model, v1 is adding support for these two new *installModes*. It should be noted that, in line with v1's no multi-tenancy policy,
+users will not be able to install the same operator multiple times, and that in future iterations of the registry bundle format will not
+include *installModes*.
+
+## Demos
+
+### SingleNamespace Install
+
+[![SingleNamespace Install Demo](https://asciinema.org/a/w1IW0xWi1S9cKQFb9jnR07mgh.svg)](https://asciinema.org/a/w1IW0xWi1S9cKQFb9jnR07mgh)
+
+### OwnNamespace Install
+
+[![OwnNamespace Install Demo](https://asciinema.org/a/Rxx6WUwAU016bXFDW74XLcM5i.svg)](https://asciinema.org/a/Rxx6WUwAU016bXFDW74XLcM5i)
+
+## Enabling the Feature-Gate
+
+!!! tip
+
+This guide assumes OLMv1 is already installed. If that is not the case,
+you can follow the [getting started](../../getting-started/olmv1_getting_started.md) guide to install OLMv1.
+
+---
+
+Patch the `operator-controller` `Deployment` adding `--feature-gates=SingleOwnNamespaceInstallSupport=true` to the
+controller container arguments:
+
+```terminal title="Enable SingleOwnNamespaceInstallSupport feature-gate"
+kubectl patch deployment -n olmv1-system operator-controller-controller-manager --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--feature-gates=SingleOwnNamespaceInstallSupport=true"}]'
+```
+
+Wait for `Deployment` rollout:
+
+```terminal title="Wait for Deployment rollout"
+kubectl rollout status -n olmv1-system deployment/operator-controller-controller-manager
+```
+
+## Configuring the `ClusterExtension`
+
+A `ClusterExtension` can be configured to install bundle in `Single-` or `OwnNamespace` mode through the
+`olm.operatorframework.io/watch-namespace: <namespace>` annotation. The *installMode* is inferred in the following way:
+
+ - *AllNamespaces*: `<namespace>` is empty, or the annotation is not present
+ - *OwnNamespace*: `<namespace>` is the install namespace (i.e. `.spec.namespace`)
+ - *SingleNamespace*: `<namespace>` not the install namespace
+
+### Examples
+
+``` terminal title="SingleNamespace install mode example"
+kubectl apply -f - <<EOF
+apiVersion: olm.operatorframework.io/v1
+kind: ClusterExtension
+metadata:
+  name: argocd
+  annotations:
+    olm.operatorframework.io/watch-namespace: argocd-watch
```

Comment:
@perdasilva based on the comment it seems that we want the same annotation name that we have in OLMV0, see: https://github.com/operator-framework/operator-controller/issues/593#issuecomment-1919858334

Reply:
I don't think we have to - the `olm.targetNamespaces` is an internal OLMv0 implementation detail. I don't think it's user facing. I also think there's value in using something different from v0 to keep both version as separate as possible the users' mind. Additionally, because it's in the plural it might give the wrong impression (i.e. that we can reference more than one). Also, this will go away once we change the `ClusterExtension` API.

Reply:
Make sense 👍